timelock-api/src/main/conjure/timelock-api.yml

```diff
@@ -20,6 +20,14 @@ types:
       base-type: any
       external:
         java: com.palantir.lock.v2.Lease
+    CommitTimestampsWithWatches:
+      base-type: any
+      external:
+        java: com.palantir.lock.watch.CommitTimestampsWithWatches
```

Comment:
as above, it's probably just easier to define this object here instead of external, unless you have strong reservations as to why not.